llvm/lib/Target/X86/X86CallingConv.td

```diff
@@ -284,7 +284,8 @@ def RetCC_X86Common : CallingConv<[
 
   // Long double types are always returned in FP0 (even with SSE),
   // except on Win64.
-  CCIfNotSubtarget<"isTargetWin64()", CCIfType<[f80], CCAssignToReg<[FP0, FP1]>>>
+  CCIfNotSubtarget<"isTargetWinOrUEFI64()",
+                    CCIfType<[f80], CCAssignToReg<[FP0, FP1]>>>
```

Comment:
Could you instead use two separate statements the same way you did for CC and avoid introducing `isTargetWinOrUEFI64()` altogether?

```suggestion
  CCIfNotSubtarget<"isTargetWin64()",
                    CCIfType<[f80], CCAssignToReg<[FP0, FP1]>>>

  CCIfNotSubtarget<"isTargetUEFI64()",
                    CCIfType<[f80], CCAssignToReg<[FP0, FP1]>>>
```

Reply:
Changed the code to drop isTargetWinOrUEFI64.